stl/inc/iterator

```diff
@@ -601,16 +601,16 @@ public:
         }
     }
 
-#if 0 // TRANSITION, VSO-1174090
+#if 0 // TRANSITION, VSO-1225825
     // clang-format off
     ~_Variantish() requires is_trivially_destructible_v<_It> && is_trivially_destructible_v<_Se> = default;
     // clang-format on
-#endif // TRANSITION, VSO-1174090
+#endif // TRANSITION, VSO-1225825
     ~_Variantish() {

```

Comment:
Should you preserve the newline you were adding here earlier?

Reply:
Yes, that was a righteous style change. I erred on the side of backing everything out here - I'll restore the newline.